### PR TITLE
Intendation fixes -- sympy/concrete/summations.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Hello.

File: `sympy/concrete/summations.py`

There were couple of `NotImplementedError` exceptions which were not intended correctly.

These have now been amended.

Thank you.

### CODE REVIEW

**Comment by jbm950** [comment]
LGTM

File: sympy/concrete/summations.py [hunk 1]
```diff
@@ -356,7 +356,7 @@ def is_convergent(self):
 
         if len(sequence_term.free_symbols) > 1:
             raise NotImplementedError("convergence checking for more that one symbol "
```
Comment (gxyd) [inline sympy/concrete/summations.py]: Can you also change `that` -> `than`?

Comment (Newman101) [inline sympy/concrete/summations.py]: Sure, I can do that. :)

**Comment by gxyd** [comment]
+1. Merging

**Pull request merged**